pkgs/by-name/wb/wb32-dfu-updater/package.nix

```diff
@@ -17,6 +17,8 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-DKsDVO00JFhR9hIZksFVJLRwC6PF9LCRpf++QywFO2w=";
   };
 
+  patches = [ ./patches/fix-cmake4-build.patch ];
```

Comment:
A comment pointing to the upstream PR would be nice. And you don't need to put patches in a dedicated directory, there already is the package directory so it is reasonably clean.

Reply:
You always give me great insights :smile: